src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -130,24 +129,42 @@ export interface IScatterPlotSampleData {
     profiledMutations?:boolean;
     mutations: AnnotatedMutation[];
     copyNumberAlterations: AnnotatedNumericGeneMolecularData[];
+    xThresholdType?: '>'|'<'|undefined; // threshold symbol for x coordinate in scatter plot
+    yThresholdType?: '>'|'<'|undefined; // threshold symbol for y coordinate in scatter plot
+    thresholdType?: '>'|'<'|undefined; // threshold symbol for continuous axis in box-scatter and waterfall plot
 }
 
-export interface IScatterPlotData extends IScatterPlotSampleData, IBaseScatterPlotData {};
-
-export interface IBoxScatterPlotPoint extends IScatterPlotSampleData {
+export interface IBoxScatterPlotPoint extends IPlotSampleData {
     category:string;
     value:number;
     jitter:number;
 }
 
+export interface IWaterfallPlotSampleData extends IPlotSampleData {
+    sortOrder:string;
+    pivotThreshold:number;
+}
+
+export interface IScatterPlotData extends IPlotSampleData, IBaseScatterPlotData {};
+export interface IWaterfallPlotData extends IWaterfallPlotSampleData, IBaseWaterfallPlotData {};
+
+export interface IAxisLogScaleParams {
+    label:string;
+    fLogScale:((x:number, offset:number)=>number);
```

Comment:
what does `offset` mean? can you add a comment

Reply:
Comment added.

Reply:
perfect, thanks